### ISSUE TITLE
How to stop a process such as 'yes'?

### ISSUE THREAD

**Issue by lucklove** [redox-os/redox#585]
**Reproduction**: [Just type 'yes' on the shell]

**Expected behavior**: [It should end when I type ctrl + 'c']

**Actual behavior**: [print 'y' all all the time].

**Comment by Vest** [comment]
I think this incident should be moved to the repository [redox-os/coreutils](https://github.com/redox-os/coreutils).

**Comment by Ticki** [comment]
No, this is the right repository. The process shouldn't handle ctrl-C.

**Comment by ca1ek** [comment]
> No, this is the right repository. The process shouldn't handle ctrl-C.

I mean, you should be able to stop stuff like this.

**Comment by Ticki** [comment]
@ca1ek Sure.

**Comment by nounoursheureux** [comment]
Do we have signals ?

**Comment by jackpot51** [comment]
No, we do not

**Comment by Ticki** [comment]
Not yet.

**Comment by bb010g** [comment]
What's the status on signals currently?
